modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ResizeHandle.ts

```diff
@@ -1,43 +1,56 @@
-import { Dragging, Focusing, Keying, SimpleSpec, Tabstopping, Tooltipping } from '@ephox/alloy';
+import { AddEventsBehaviour, AlloyComponent, AlloyEvents, Dragging, Focusing, Keying, SimpleSpec, SystemEvents, Tabstopping, Tooltipping } from '@ephox/alloy';
 import { Optional } from '@ephox/katamari';
-import { SugarPosition } from '@ephox/sugar';
+import { Attribute, SugarPosition } from '@ephox/sugar';
 
 import Editor from 'tinymce/core/api/Editor';
 
 import * as Options from '../../api/Options';
 import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
 import * as Icons from '../icons/Icons';
-import { resize, ResizeTypes } from '../sizing/Resize';
+import * as Resize from '../sizing/Resize';
 
-const getResizeType = (editor: Editor): ResizeTypes => {
+const getResizeType = (editor: Editor): Resize.ResizeTypes => {
   const resize = Options.getResize(editor);
   if (resize === false) {
-    return ResizeTypes.None;
+    return Resize.ResizeTypes.None;
   } else if (resize === 'both') {
-    return ResizeTypes.Both;
+    return Resize.ResizeTypes.Both;
   } else {
-    return ResizeTypes.Vertical;
+    return Resize.ResizeTypes.Vertical;
   }
 };
 
-const keyboardHandler = (editor: Editor, resizeType: ResizeTypes, x: number, y: number): Optional<boolean> => {
+const getAriaValuetext = (dimensions: Resize.EditorDimensions, resizeType: Resize.ResizeTypes): string => {
+  return resizeType === Resize.ResizeTypes.Both
```

Comment:
These need to be translated to use the backstage translate on these also add a ticket to stick them into the pots and the dynamic values need to be `{0}` type variables for the translation.